messaging-experiments.yaml

```diff
@@ -0,0 +1,34 @@
+- id: aw_pull_factor_privacy_1
+  filter_expression: localeLanguageCode == "en" && browserSettings.update.channel ==
+    "release" && firefoxVersion >= 75 && [userId, "aw_pull_factor_privacy_1"]|bucketSample(0,
+    2000, 10000)
+  arguments:
+    slug: aw_pull_factor_privacy_1
+    branches:
+      - slug: control
+        ratio: 1
+        value: {}
+      - slug: variant_1
+        ratio: 1
+        value:
+          title: Welcome to Firefox. Fast, safe, private.
+          cards:
+            - PERSONAL_DATA_PROMISE_CARD
```

Comment:
These aren't real card values, can you just remove the `card` property in both branches for now?